docs/quality-assurance/bdd.md

```diff
@@ -1,82 +1,82 @@
-# Behaviour-Driven Design (BDD)
+# Behaviour Driven Development (BDD)
+
+## What is BDD
 
 > "An approach that enhances communication between stakeholders and project team members by expressing product needs as concrete examples"
 
-This page contains information about BDD and how it can be used to aid how we develop software at UKHO.
+BDD:
 
-## What is BDD
+* Is a **process** to enable the entire team to better understand stories before they are played
+* Enables structured **collaboration** between Product Owners (proxy for the customer), developers and testers to gain an agreed understanding
+* Uses **rules and examples** to illustrate this understanding
 
-* A process to enable the entire team to better understand stories before they are played
-* BDD is about _collaboration_ - Product Owners (proxy for the customer), developers and testers communicating to gain an agreed understanding
-* Uses rules and examples to illustrate this understanding
-* When stories are played there is less ambiguity leading to better software development
-* BDD is NOT:
-    * Testing
-    * SpecFlow or Cucumber
-    * Just automated tests (tests are a downstream benefit)
+BDD is NOT:
 
-## Aims of BDD
+* Testing
+* Gherkin
+* SpecFlow or Cucumber
+* Just automated tests (tests are a downstream benefit)
 
-BDD aims to make the development of software more successful by:
+## Benefits of BDD
 
-* Incorporating diverse perspectives when defining requirements
-* Creating a shared understanding in a delivery team of the required product changes
-* Simplifying and shortening team Scrum events (refinement and planning)
-* Ensures only the functionality to meet requirements is developed
-* Reducing the amount of re-work required in the maintenance phase (by making the original change more complete)
+* Creates a shared understanding in the team of user requirements
+* Ensures all perspectives are considered when defining requirements
+* Produces better software that matches user needs first time
+
+![bdd tree swing](images/bdd_tree_swing.png)
```

Comment:
Would this image be better higher up? Maybe in the "What is BDD" bit

Reply:
I like it where it is but if others think it's better further up then it can be moved